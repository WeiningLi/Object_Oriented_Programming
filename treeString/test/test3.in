ra "Hello "
^a6 "this "
^a11 "is "
^a14 "used "
^a19 "to "
^a22 "test "
^a27 "^ "
^a29 "a"
^a30 "b "
^a32 "c"
^a33 "d"
^a34 "e"
^a35 "f"
^a36 "g"
^a37 "kk   "
cba
da
pa
ia30?
+aab
pa
da
q
