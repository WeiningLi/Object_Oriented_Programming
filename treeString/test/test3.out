"Hello " size: 6
 "this " size: 5
  "is " size: 3
   "used " size: 5
    "to " size: 3
     "test " size: 5
      "^ " size: 2
       "a" size: 1
        "b " size: 2
         "c" size: 1
          "d" size: 1
           "e" size: 1
            "f" size: 1
             "g" size: 1
              "kk   " size: 5
Hello this is used to test ^ ab cdefgkk   
Hello this is used to test ^ a? cdefgkk   Hello this is used to test ^ ab cdefgkk   
"kk   " size: 42
 "Hello " size: 6
  "this " size: 5
   "is " size: 3
    "used " size: 5
     "to " size: 3
      "test " size: 5
       "^ " size: 2
        "a" size: 1
         "? " size: 2
          "c" size: 1
           "d" size: 1
            "e" size: 1
             "f" size: 1
              "g" size: 1
 "Hello " size: 6
  "this " size: 5
   "is " size: 3
    "used " size: 5
     "to " size: 3
      "test " size: 5
       "^ " size: 2
        "a" size: 1
         "b " size: 2
          "c" size: 1
           "d" size: 1
            "e" size: 1
             "f" size: 1
              "g" size: 1
               "kk   " size: 5
